### PR TITLE
Update query tip JSON in documentation

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: doc/reference/building-and-signing-tx.md [hunk 1]
```diff
@@ -67,15 +67,16 @@ When building and submitting a transaction in the shelley era you need to check
 Look for the value of `SlotNo`
 
     {
-        "blockNo": 16829,
-        "headerHash": "3e6f59b10d605e7f59ba8383cb0ddcd42480ddcc0a85d41bad1e4648eb5465ad",
-        "slotNo": 369200
+        "epoch": 259,
```
Comment (Jimbo4350) [inline doc/reference/building-and-signing-tx.md:70]: :+1:

File: doc/reference/shelley-genesis.md [hunk 2]
```diff
@@ -1430,9 +1437,9 @@ Usage: cardano-cli transaction build-raw [--byron-era | --shelley-era |
 
 Available options:
   --byron-era              Specify the Byron era
-  --shelley-era            Specify the Shelley era (default)
+  --shelley-era            Specify the Shelley era
   --allegra-era            Specify the Allegra era
-  --mary-era               Specify the Mary era
+  --mary-era               Specify the Mary era (default)
```
Comment (Jimbo4350) [inline doc/reference/shelley-genesis.md:1442]: :+1:

File: doc/stake-pool-operations/KES_period.md [hunk 3]
```diff
@@ -25,21 +25,22 @@ We check the current tip of the blockchain:
...
+        "slot": 26633911,
+        "block": 5580350
     }
 
-In this example, we are currently in slot 8520857, and we know from the genesis file that one period lasts for 129600 slots. So we calculate the current period by
+In this example, we are currently in slot 26633911, and we know from the genesis file that one period lasts for 129600 slots. So we calculate the current period by
 
-    expr 8520857 / 129600
-    > 65
+    expr 26633911 / 129600
```
Comment (Jimbo4350) [inline doc/stake-pool-operations/KES_period.md:36]: Is this calculation valid? For a pure Shelley testnet it would be but for mainnet?

**Review by Jimbo4350 — APPROVED** [review]
LGTM, just a question about the KES period calculation

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**